lib/compat/wordpress-6.3/class-gutenberg-rest-templates-controller-6-3.php

```diff
@@ -100,7 +100,7 @@ public function prepare_item_for_response( $item, $request ) { // phpcs:ignore V
 	/**
 	 * Adds revisions to links.
 	 *
-	 * @since 6.3.0
+	 * @since 6.2.0
```

Comment:
Nit: It seems that the revision links code never made it in 6.2.0 from the sounds of it: https://github.com/WordPress/gutenberg/pull/45215#issuecomment-1592704026, so should this still be `6.3.0`?

Reply:
I have no idea what that is doing there. Thanks for flagging it. 
Brain meltdown.

Reply:
Oh, bad rebase from https://github.com/WordPress/gutenberg/pull/51362/files#diff-4592dfe24434861ba8dc2c1dbd3b6c83528b1e62050e9bb834e7d9f256a83058L101